Server/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.h

```diff
@@ -47,6 +47,7 @@ class CLuaPlayerDefs : public CLuaDefs
     LUA_DECLARE(GetPlayerVersion);
     LUA_DECLARE(GetPlayerACInfo);
     static unsigned int GetPlayerScriptDebugLevel(CPlayer* const player);
+    static bool HasPlayerSpawned(CPlayer* player);
```

Comment:
player can be const?

Reply:
When I marked any pointer `const`, ArgumentParser rejected these functions and threw a compile time error